client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
+func (client *Client) Package(pkgID string) (Package, error) {
```

Comment:
Why is it returning Package by value here? It doesn't feel like those will generally have value semantics.